oci/manager.go

```diff
@@ -216,80 +236,183 @@ func (manager *ociManager) Catalog(ctx context.Context, qopts ...packmanager.Que
 		}
 	}
 
-	// Access local images that are available on the host
-	manifests, err := handle.ListManifests(ctx)
-	if err != nil {
-		return nil, err
-	}
-
-	for _, manifest := range manifests {
-		// Check if the OCI image has a known annotation which identifies if a
-		// unikernel is contained within
-		if _, ok := manifest.Annotations[AnnotationKernelVersion]; !ok {
-			log.G(ctx).
-				WithField("ref", manifest.Config.Digest.String()).
-				Trace("skipping non-unikernel digest")
-			continue
+	if !query.Multiple() {
```

Comment:
Can you explain what the difference between when `query.Multiple` is `true` and `false`?

Reply:
I wanted to have an implementation-agnostic name

in our case:
* false: push/pull single manifest
* true: push/pull index containing multiple manifests

Reply:
Ah i see. I guess it should then be renamed to `AllTargets` (similar to `ctr i pull --all-platforms`).